proxied.d.ts

```diff
@@ -1167,6 +1174,8 @@ export interface InputFileProxy<F> {
       tgs_sticker?: F;
       /** WEBM video with the sticker, uploaded using multipart/form-data. See https://core.telegram.org/stickers#video-sticker-requirements for technical requirements */
       webm_sticker?: F;
+      /** Type of stickers in the set, pass “regular” or “mask”. Custom emoji sticker sets can't be created via the Bot API at the moment. By default, a regular sticker set is created. */
+      sticker_type?: "regular" | "mask";
```

Comment:
They have also removed `contains_masks` (backward compatibility, yes) property from `createNewStickerSet`.

Reply:
Again, my bad, I will bring it back!

Reply:
No 👀, don't bring them back. You have to remove the `contains_masks`.

Reply:
Okay 🙂